paasta_tools/autoscaling_lib.py

```diff
@@ -753,16 +755,17 @@ def scale_aws_spot_fleet_request(resource, current_capacity, target_capacity, so
                 break
             # The start time of the maintenance window is the point at which
             # we giveup waiting for the instance to drain and mark it for termination anyway
-            start = int(time.time() + CLUSTER_DRAIN_TIMEOUT)
+            start = int(time.time() + CLUSTER_DRAIN_TIMEOUT) * 1000000000  # nanoseconds
```

Comment:
There is `seconds_to_nanoseconds` in `paasta_tools.paasta_maintenance`, but I guess importing and doing the function call might be overkill for something this simple.